broker/src/test/java/org/apache/rocketmq/broker/filter/MessageStoreWithFilterTest.java

```diff
@@ -238,7 +238,7 @@ public void testGetMessage_withFilterBitMapAndConsumerChanged() {
 
             // sleep to wait for consume queue has been constructed.
             try {
-                Thread.sleep(1000);
+                Thread.sleep(2001);
```

Comment:
Do we have any method to ensure CQ has been constructed? Sleep is not a good practice~